doc/releases/migration-guide-3.6.rst

```diff
@@ -465,6 +465,64 @@ zcbor
   :c:func:`zcbor_log`, and :c:func:`zcbor_trace` with no parameters is gone, and in its place are
   :c:func:`zcbor_trace_file` and :c:func:`zcbor_trace`, both of which take a ``state`` parameter.
 
+CFB
+===
```

Comment:
This needs to go into the migration guide for v3.7.0 instead.

Reply:
Sorry for my late response.
Update done.